scripts/extraOptions

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Copyright (C) Lightbend Inc. <https://www.lightbend.com>
+
+# Lib for CI scripts
+# copied and adapted from https://github.com/lagom/lagom/blob/1738a589816c6b12e4f9bcc05b7cba4d184da3b6/bin/scriptLib#L5
+
+set -e
+set -o pipefail
+
+EXTRA_OPTS=("")
+
+# Check if it is a scheduled build
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+    ## Search for snapshots is harcdoded to scala 2.13
+    ## Queries differ because the artifact names changed during the 2.6 milestones phase.
+    AKKA_25_SNAPSHOT=$(curl -s https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.13/ | grep -oEi '2\.5-[0-9]{8}-[0-9]{6}' | sort | tail -n 1)
```

Comment:
sort -V?

Reply:
Yeah, `-V` wouldn't harm but seem to produce the exact same output:

```
$ curl -s https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.13/ | grep -oEi '2\.5-[0-9]{8}-[0-9]{6}' | sort | tail -n 5
2.5-20200130-220103
2.5-20200204-220059
2.5-20200204-220059
2.5-20200210-220117
2.5-20200210-220117
$ curl -s https://repo.akka.io/snapshots/com/typesafe/akka/akka-actor_2.13/ | grep -oEi '2\.5-[0-9]{8}-[0-9]{6}' | sort -V | tail -n 5
2.5-20200130-220103
2.5-20200204-220059
2.5-20200204-220059
2.5-20200210-220117
2.5-20200210-220117
```

TBH, I didn't write the bash spell, I just copy/pasted it from the lagom/lagom  `.travis.yml` as I knew that worked already. :-)